lib/scsbxml_fetcher.rb

```diff
@@ -14,11 +16,13 @@ class SCSBXMLFetcher
   #    This is the output of BarcodeToCustomerCodeMapper#barcode_to_customer_code_mapping
   def initialize(options = {})
     @token = nil
+    @errors = {}
     @oauth_url = options[:oauth_url]
     @oauth_key = options[:oauth_key]
     @oauth_secret = options[:oauth_secret]
     @platform_api_url = options[:platform_api_url]
-    @barcode_to_customer_code_mapping = options[:barcode_to_customer_code_mapping]
+    @barcode_to_customer_code_mapping =
```

Comment:
I know this is probably from a Rubocop comment but it feels weird to break up an assignment.
Am I being a weirdo?

Reply:
yes it is from Rubocop who complained the line is too long! It is OK I can put it back.